src/style/shadows.scss

```diff
@@ -110,94 +110,105 @@
 
-    --shadow-depth-64: 0 #{functions.pxToRem(25.6)} #{functions.pxToRem(57.6)} 0
-            rgba(var(--color-black), 0.22),
-        0 #{functions.pxToRem(4.8)} #{functions.pxToRem(14.4)} 0
-            rgba(var(--color-black), 0.58);
-
-    --shadow-inflated-8: #{functions.pxToRem(-2)} #{functions.pxToRem(-2)} #{functions.pxToRem(
-                6
-            )} rgba(var(--color-black), 0.21),
-        #{functions.pxToRem(-4)} #{functions.pxToRem(-4)} #{functions.pxToRem(
-                10
-            )} rgba(var(--color-white), 0.08),
-        #{functions.pxToRem(4)} #{functions.pxToRem(4)} #{functions.pxToRem(10)}
-            rgba(var(--color-black), 0.46),
-        #{functions.pxToRem(1)} #{functions.pxToRem(1)} #{functions.pxToRem(10)}
-            rgba(var(--color-white), 0.16),
-        -1px -1px 0 1px rgba(var(--color-black), 0.37) inset,
-        0 0 1px 1px rgba(var(--color-white), 0.2) inset;
-
-    --shadow-inflated-16: #{functions.pxToRem(-3)} #{functions.pxToRem(-3)} #{functions.pxToRem(
-                6
-            )} rgba(var(--color-black), 0.21),
-        #{functions.pxToRem(-6)} #{functions.pxToRem(-6)} #{functions.pxToRem(
-                16
-            )} rgba(var(--color-white), 0.1),
-        #{functions.pxToRem(6)} #{functions.pxToRem(6)} #{functions.pxToRem(16)}
-            rgba(var(--color-black), 0.46),
-        #{functions.pxToRem(2)} #{functions.pxToRem(2)} #{functions.pxToRem(10)}
-            rgba(var(--color-white), 0.16),
-        -1px -1px 0 1px rgba(var(--color-black), 0.37) inset,
-        0 0 1px 1px rgba(var(--color-white), 0.2) inset;
-
-    --shadow-inflated-64: #{functions.pxToRem(-3)} #{functions.pxToRem(-3)} #{functions.pxToRem(
-                6
-            )} rgba(var(--color-black), 0.21),
-        #{functions.pxToRem(-10)} #{functions.pxToRem(-10)} #{functions.pxToRem(
-                30
-            )} rgba(var(--color-white), 0.2),
-        #{functions.pxToRem(10)} #{functions.pxToRem(10)} #{functions.pxToRem(
-                36
-            )} rgba(var(--color-black), 0.46),
-        #{functions.pxToRem(2)} #{functions.pxToRem(2)} #{functions.pxToRem(10)}
-            rgba(var(--color-white), 0.16),
-        -1px -1px 0 1px rgba(var(--color-black), 0.37) inset,
-        0 0 1px 1px rgba(var(--color-white), 0.2) inset;
-}
+// ⚠️ This section below is commented out.
+// It's contents are moved to src/style/color-palette-extended.css
+//
+// Because we don't want to give dark-mode-specific shadows to those
+// clients that only import src/style/color-palette-extended-light-mode-only.css
+// Because if their OS is in dark mode, not their app,
+// The shadows will get rendered too dark, due to `@media (prefers-color-scheme: dark)`
+// rule from the mixin. The day we generate `.css` files from these `.scss` files,
+// we can do it differently.
+
+// @include mixins.in(dark-mode) {
+//     --button-shadow-normal: 0 #{functions.pxToRem(1.5)} #{functions.pxToRem(
```

Comment:
Do we need to leave this commented out code here?

Reply:
Yeah, the PR works, so I'll approve and merge it 👍

Then we can enable releases again! 🎉